packages/es-components/src/components/controls/answer-group/AnswerGroup.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const AnswerSet = styled.div`
+  display: flex;
+  flex-wrap: nowrap;
+
+  @media (min-width: ${props => props.theme.screenSize.desktop}) {
+    float: left;
```

Comment:
Layout will be a little easier if instead of using a float here, we make the AnswerButtons `flex-grow: 0` at desktop size.

Reply:
fixed